plugins/micromega/g_zify.mlg

```diff
@@ -40,6 +40,8 @@ END
 TACTIC EXTEND TRANS
 | [ "zify_op"  ] -> { Zify.zify_tac }
 | [ "saturate"  ] -> { Zify.saturate }
+| [ "assert_inj" constr(t)] -> { Zify.assert_inj t }
+| [ "iter_let" tactic(t)] -> { Zify.iter_let t }
```

Comment:
If possible, can you simply declare this as a pure Ltac entry rather than a `TACTIC EXTEND`? The latter also adds parsing rules, and I bet several people have tactics called `iter_let`. Alternatively, you could put the declaration inside a Coq module not meant to be imported.

Reply:
I don't know how to implement what you propose.
I have prefixed the tactics by "zify_". That should at least minimise the risk of clashes.
Also, in the last commit, iter_let is not exported, there is zify_elim_let.

Reply:
I'll take care of that in the cleanup PR.